core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2727,6 +2726,11 @@ object SparkContext extends Logging {
         scheduler.initialize(backend)
         (backend, scheduler)
 
+      case zkUrl if zkUrl.startsWith("zk://") =>
+        logWarning("Master URL for a multi-master Mesos cluster managed by ZooKeeper should be " +
```

Comment:
Nitpicking, but I would be more explicit: "This will stop working in Spark 2.0".